string name
geometry_msgs/Pose pose
float32 fitness_score
int32 type
geometry_msgs/Polygon planar